templates/vault_systemd.service.j2

```diff
@@ -25,7 +25,9 @@ PrivateTmp=yes
 PrivateDevices=yes
 SecureBits=keep-caps
 Capabilities=CAP_IPC_LOCK+ep
+{% if systemd_version | version_compare('230', '>=') %}
```

Comment:
It may be safer to cast this variable to int, but I don't know how to do it when registering it https://github.com/brianshumate/ansible-vault/pull/68/files#diff-2444ad0870f91f17ca6c2a5e96b26823R139